airbyte-config-oss/init-oss/src/main/resources/seed/source_definitions.yaml

```diff
@@ -1676,14 +1676,6 @@
   icon: qualaroo.svg
   sourceType: api
   releaseStage: alpha
-- name: QuickBooks
-  sourceDefinitionId: 29b409d9-30a5-4cc8-ad50-886eb846fea3
-  dockerRepository: airbyte/source-quickbooks-singer
-  dockerImageTag: 0.1.5
-  documentationUrl: https://docs.airbyte.com/integrations/sources/quickbooks-singer
-  icon: qb.svg
-  sourceType: api
-  releaseStage: alpha
```

Comment:
There's still something wrong here - the connector should not be removed from the definitions

Reply:
@evantahler this connector has never been enabled in cloud and the new one has a different name. Can you suggest what I should do to delete the old one and add the new connector?

Reply:
Shouldn't this low-code connector be using the same sourceDefinitionId and name?

Reply:
It seems weird to me that the lowcode connector would be named `source-quickbooks-singer`

Reply:
Keep the ID, but change the name! That's OK!

Reply:
Tagging @pedroslopez (who is out until next week) - What's the right thing to do here?  We want to keep the same soureId so that people can upgrade, but we want to change the name of the connector

Reply:
In this case, we should:
1) change the name first > merge
2) migrate to low-code (which is done) > merge
3) have a bunch of backward compatibility bypass for CAT between 1 and 2 to cover the name change (this will touch lots of files not just a single line, if we want to have a name changed everywhere, not just spec)

Apologies, but quickbooks-singer was not even worked, as far as i know, so it's a question what is the pool of users affected? @evantahler do you have possibility to check this? 
If there is any, we should probably change the name > migrate to low-code, otherwise - there is nothing to do but remove sourceDefinition of existing QuickBooks source, and replace with the new one for low-code. WDYT?

There is option number 3, of course: 
Release this change as separate connector (alpha) as low code connector and forget about singer-one for now. Announce the breaking change for the users > start using new implementation > come back and remove the old one. To much effort.

Reply:
I think we shouldn't worry too much about existing users. It's dubious this connector ever really worked.